environment.yml

```diff
@@ -2,28 +2,21 @@ name: ticoi
 channels:
   - conda-forge
 dependencies:
-  - python=3.11
-  - aiohttp
   - dask
-  - fiona=1.9
-  - gdal
-  - geopandas>=0.10.0
-  - h5netcdf
-  - h5py>=3.11
+  - geopandas
```

Comment:
you might want to pin some versions, no? Maybe not too strict, but minimum versions of certain packages at least.